deploy/helm_charts/dc_website/templates/deployment.yaml

```diff
@@ -130,4 +130,73 @@ spec:
               port: 6060
           ports:
             - containerPort: 6060
-        {{- end }}
\ No newline at end of file
+        {{- end }}
+
+{{- if .Values.nodejs.enabled }}
+---
+  
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: website-nodejs-app
+  namespace: {{ .Values.namespace.name }}
+spec:
+  replicas: {{ .Values.nodejs.replicas}}
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      # maximum number of Pods that can be created over the desired number of Pods
+      # 25% of default pods (+1 in case default < 4)
+      maxSurge: {{ div .Values.nodejs.replicas 4 | add 1 }}
+      # Maximum number of pods that can be unavailable during the update process
+      # 25% of default pods (+1 in case default < 4)
+      maxUnavailable: {{ div .Values.nodejs.replicas 4 | add 1 }}
+  selector:
+    matchLabels:
+      app: website-nodejs-app
+  template:
+    metadata:
+      labels:
+        app: website-nodejs-app
+      annotations:
+        timestamp: {{ now | quote }}
+    spec:
+      serviceAccountName: {{ .Values.serviceAccount.name }}
+      containers:
+        - name: nodejs
+          image:  "gcr.io/{{ .Values.website.image.project }}/datacommons-nodejs:{{ .Values.website.image.tag }}"
+          imagePullPolicy: {{ .Values.website.image.pullPolicy }}
+          args: []
+          ports:
+            - containerPort: 8080
+          resources:
+            limits:
+              memory: "3G"
+            requests:
+              memory: "3G"
```

Comment:
I think 1 or 2G memory should be more than enough, since all this does is to render an svg?

Reply:
it will eventually be multiple svgs and multiple data csvs, but you're right, don't need 3G. Updated